src/camera/src/Camera.cpp

```diff
@@ -8,6 +8,13 @@
 
 Camera *Camera::_camera = nullptr;
 
+Camera::Camera()
+    : _bounding_x(3.f)
```

Comment:
Tune the values so they would correspond to the original ones. Right now the bounding box is too big.

Reply:
It's tuned to 0.5% of the original values, and for the time being it's difficult to be pixel-perfect.